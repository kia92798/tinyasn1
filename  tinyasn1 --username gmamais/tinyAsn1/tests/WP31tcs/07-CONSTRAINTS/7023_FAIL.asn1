--FSemantic Error: SIZE constraint in integer type
--EMyMod&nbsp;&nbsp;DEFINITIONS ::=
--EBEGIN
--E&nbsp;&nbsp;Int ::= INTEGER (SIZE(0..4))
--EEND

MyMod	DEFINITIONS ::=
BEGIN
	Int ::= INTEGER (SIZE(0..4))
END
