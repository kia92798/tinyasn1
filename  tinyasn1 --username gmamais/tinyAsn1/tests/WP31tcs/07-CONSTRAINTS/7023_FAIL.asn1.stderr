Error in Line:9, col:18 . This type of constraint 'SIZE'cannot appear under INTEGER
